crates/bevy_animation/src/lib.rs

```diff
@@ -826,55 +1094,51 @@ pub fn advance_animations(
 }
 
 /// A system that modifies animation targets (e.g. bones in a skinned mesh)
-/// according to the currently-playing animation.
+/// according to the currently-playing animations.
 pub fn animate_targets(
     clips: Res<Assets<AnimationClip>>,
     graphs: Res<Assets<AnimationGraph>>,
     players: Query<(&AnimationPlayer, &Handle<AnimationGraph>)>,
     mut targets: Query<(
-        Entity,
-        &AnimationTarget,
```

Comment:
Will it be faster if we keep the `&AnimationTarget` here? `QueryState` cache the matehed archetype，if we put `&AnimationTarget` here will reduce search scope with `QueryState`'s help。